.github/ISSUE_TEMPLATE/portfolio-review.yml

```
@@ -13,6 +13,13 @@ body:
       placeholder: "e.g.: https://<your-username>.github.io/portfolio"
     validations:
       required: true
+  - type: input
+    id: link
```

`id` needs to be unique

```suggestion
    id: repo
```